### PR TITLE
Feature/suggestion interval

### PR DESCRIPTION
**Pull request by emrecalik**
Suggested users and articles are rendered on homepage.

### CODE REVIEW

File: Project/Wikode/templates/wikodeApp/homePage.html [hunk 1]
```diff
@@ -61,69 +61,38 @@
...
-                                                    <p class="no-border">
-                                                        <small> Rodriguez-Morales AJ, Gallego V, Escalera-Antezana
-                                                            JP </small>
-                                                    </p>
-                                                </div>
-                                            </li>
+                                            {% for articleSuggestionDTO in articleSuggestionDTOList %}
+                                                <li class="list-group-item">
+                                                    <div class="media-body">
+                                                        <!--This redirection here is dummy for now, we will map it to real articles-->
```
Comment (active-sludge) [inline Project/Wikode/templates/wikodeApp/homePage.html:67]: Delete pls.

File: Project/Wikode/wikodeApp/utils/suggestionManager.py [hunk 2]
```diff
@@ -136,19 +116,17 @@ def get_user_suggestion(self):
...
         article_list = self.get_article_id_list_from_followee(activity_type=1)
         return article_list
 
-
     # Gets the most viewed article (1)
     # chekcs if the owner has already seen it
     def get_most_viewed_article(self):
-        most_common = Activity.objects.filter(activity_type=1,target_type=3).annotate(mc=Count('target_id')).order_by('-mc')[0]
+        most_common = \
+            Activity.objects.filter(activity_type=1, target_type=3).annotate(mc=Count('target_id')).order_by('-mc')[0]
```
Comment (active-sludge) [inline Project/Wikode/wikodeApp/utils/suggestionManager.py:129]:
```
most_common = Activity.objects.filter(activity_type=1, target_type=3)
                                          .annotate(mc=Count('target_id'))
                                          .order_by('-mc')[0]
```

gibi yapabilir miyiz?

File: Project/Wikode/wikodeApp/utils/suggestionManager.py [hunk 3]
```diff
@@ -217,15 +192,15 @@ def get_other_tagged_articles(self):
...
-
     # Returns an article list from followee
     # depending on activity type
     def get_article_id_list_from_followee(self, activity_type):
         article_id_list = []
         if self.followees:
             for followee in self.followees:
                 if followee:
-                    followee_list = Activity.objects.filter(user_id=followee[0], activity_type=activity_type, target_type=3)
+                    followee_list = Activity.objects.filter(user_id=followee[0], activity_type=activity_type,
```
Comment (active-sludge) [inline Project/Wikode/wikodeApp/utils/suggestionManager.py:202]:
```
followee_list = Activity.objects.filter(user_id=followee[0], 
                                                              activity_type=activity_type,
                                                              target_type=3
)
```

**Review by active-sludge — APPROVED** [review]

**Review by kenankural — APPROVED** [review]
Eline sağlık abi

**Pull request merged**